service/matching/version_rule_helpers.go

```diff
@@ -73,6 +77,7 @@ var (
 	errExceedsMaxRuleChain = func(cnt, max int) error {
 		return serviceerror.NewFailedPrecondition(fmt.Sprintf("update exceeds number of chained redirect rules permitted in namespace (%v/%v)", cnt, max))
 	}
+	errUnversionedRedirectRuleTarget = serviceerror.NewInvalidArgument(fmt.Sprintf("the unversioned build id '%s' cannot be the target of a redirect rule", unversionedBuildId))
```

Comment:
This is just an empty string, I think the empty quotes are confusing.

```suggestion
	errUnversionedRedirectRuleTarget = serviceerror.NewInvalidArgument(fmt.Sprintf("the unversioned build id cannot be the target of a redirect rule", unversionedBuildId))
```